src/common/state/table.h

```diff
@@ -44,6 +44,11 @@ typedef struct {
   table_fail_callback fail_callback;
 } RetryInfo;
 
+static const RetryInfo heartbeat_retry = {
```

Comment:
Oh, we're using it in `db_client_table.cc`, but yeah, we can define it there instead. I don't want to pass in `NULL` since the default retry is to try things infinitely (we should just try heartbeats once).